cartoframes/context.py

```diff
@@ -193,6 +193,23 @@ def write(self, df, table_name, temp_dir='/tmp', overwrite=False,
                        base_url=self.base_url,
                        table_name=final_table_name))
 
+    def delete(self, table_name):
+        """Delete table
+
+        Args:
+            table_name (str): Table name to delete
+
+        Returns:
+            None
+        """
+        try:
+            self.auth_client.send('api/v1/viz/{table_name}'.format(table_name=table_name),
+                                  http_method='DELETE')
+        except CartoException as err:
```

Comment:
It might be good to pass the text in `err` to the warning to help someone debug if something weird arises.